ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/components/CustomTimeText.kt

```diff
@@ -61,6 +75,24 @@ fun CustomTimeText(
                     )
                 }
             } else null,
+            trailingCurvedContent = if (showWarning) {
```

Comment:
How about adding a comment that trailing text is against UX guidance for real apps, but is convenient here as a reminder not to use Debug builds for performance testing?

Reply:
Do we have a link to the guidance?

Reply:
Will update once we find it.